### ISSUE TITLE
Can't run on CMD or Windows 10 GUI

### ISSUE THREAD

**Issue by DaniyarNauryz** [odedshimon/BruteShark#137]
My Windows 10 GUI of BruteShark doesn't start at all. CMD works, but I
couldn't find a list of CMD commands for BS.

How to extract passwords with its help? I send you a screenshot of my
attempt to extract a password.
![Dan bruteshark](https://github.com/odedshimon/BruteShark/assets/68249290/653dd75d-8fb3-4ae3-9309-ce2f40ce15e1)

**Comment by sipowlad** [comment]
@DaniyarNauryz have you satisfied all prerequisites specified on https://github.com/odedshimon/BruteShark#windows ?
Installation of https://dotnet.microsoft.com/en-us/download/dotnet/3.1/runtime fixed the issue for me.

**Comment by JMVRy** [comment]
I had the same issue, and was confused for a bit. Turns out, I had the CLI version of the .NET runtime installed, but not the "Desktop" version. Installing that allowed it to open instantly.

**Comment by DaniyarNauryz** [comment]
> I had the same issue, and was confused for a bit. Turns out, I had the CLI version of the .NET runtime installed, but not the "Desktop" version. Installing that allowed it to open instantly.

I have managed to run its CLI version through CMD.

**Comment by DaniyarNauryz** [comment]
> @DaniyarNauryz have you satisfied all prerequisites specified on https://github.com/odedshimon/BruteShark#windows ? Installation of https://dotnet.microsoft.com/en-us/download/dotnet/3.1/runtime fixed the issue for me.

No. I cannot run its GUI version even after switching to a desktop runtime.

**Comment by DaniyarNauryz** [comment]
After running multiple CLI modules on all PCAP files in a directory, I have yielded ZERO output. Just an empty results' folder.
